test/e2e/pod_top_test.go

```diff
@@ -119,13 +120,15 @@ var _ = Describe("Podman top", func() {
 		_, ec, podid := podmanTest.CreatePod("")
 		Expect(ec).To(Equal(0))
 
-		session := podmanTest.Podman([]string{"run", "-d", "--pod", podid, ALPINE, "top", "-d", "2"})
+		session := podmanTest.Podman([]string{"run", "-d", "--pod", podid, ALPINE, "sleep", "2000"})
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(0))
 
-		session = podmanTest.Podman([]string{"run", "-d", "--pod", podid, ALPINE, "top", "-d", "2"})
+		session = podmanTest.Podman([]string{"run", "-d", "--pod", podid, ALPINE, "sleep", "2000"})
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(0))
+		// Make containers have time to start
+		time.Sleep(1 * time.Second)
```

Comment:
we should be using podman wait here if you feel this is really the problem

Reply:
I think we want to run while the containers are alive, not when they're both stopped